### ISSUE TITLE
Unable to add accessibility attrs to yew_nested_router::components::Link 

### ISSUE THREAD

**Issue by phill-85** [ctron/patternfly-yew#51]
We should be able to set the aria attributes on the Breadcrumb component, but cannot in all variants.

See the `breadcrumb_router_item` function in `components/breadcrumb/router.rs`

We should investigate if this is an issue elsewhere.
